### ISSUE TITLE
Connection interruption with Asus AX5400 Router via WiFi

### ISSUE THREAD

**Issue by A123670011** [abrenoch/hyperion-android-grabber#178]
I have changed my router to a Asus TUF AX5400 router. 
Now I have short connection interruptions each 5 seconds to my hyperion session. 

If I connect the shield via LAN it works fine. 
With my existing TP-LINK router it works also fine. This behaviour is only with the ASUS router connected via WiFi with 2,4GHz and 5GHz. 

Find below an example of the log file:
2022-01-27T18:45:20.116Z [ PROTOSERVER] (DEBUG) (ProtoServer.cpp:68:newConnection()) New connection from ::ffff:192.168.1.231
2022-01-27T18:45:20.125Z [ HYPERION] (DEBUG) (PriorityMuxer.cpp:165:registerInput()) Register new input 'Proto@::ffff:192.168.1.231/PROTOSERVER' with priority 100 as inactive
2022-01-27T18:45:20.127Z [ HYPERION] (DEBUG) (PriorityMuxer.cpp:271:setInputImage()) Priority 100 is now active
2022-01-27T18:45:20.127Z [ HYPERION] (DEBUG) (PriorityMuxer.cpp:374:setCurrentTime()) Set visible priority to 100
2022-01-27T18:45:20.128Z [ HYPERION] (DEBUG) (Hyperion.cpp:571:handleSourceAvailability()) priority[100], previousPriority[255]
2022-01-27T18:45:20.129Z [ HYPERION] (DEBUG) (Hyperion.cpp:582:handleSourceAvailability()) new source available -> Resume output processing and switch LED-Device on
2022-01-27T18:45:26.734Z [ PROTOSERVER] (DEBUG) (ProtoServer.cpp:68:newConnection()) Previous line repeats 1 times
2022-01-27T18:45:26.734Z [ PROTOSERVER] (DEBUG) (ProtoClientConnection.cpp:76:disconnected()) Socket Closed
2022-01-27T18:45:26.735Z [ HYPERION] (DEBUG) (PriorityMuxer.cpp:292:clearInput()) Removed source priority 100
2022-01-27T18:45:26.735Z [ HYPERION] (DEBUG) (PriorityMuxer.cpp:374:setCurrentTime()) Set visible priority to 255
2022-01-27T18:45:26.736Z [ HYPERION] (DEBUG) (Hyperion.cpp:571:handleSourceAvailability()) priority[255], previousPriority[100]
2022-01-27T18:45:26.736Z [ HYPERION] (DEBUG) (Hyperion.cpp:574:handleSourceAvailability()) No source left -> Pause output processing and switch LED-Device off
2022-01-27T18:45:31.738Z [ PROTOSERVER] (DEBUG) (ProtoServer.cpp:68:newConnection()) New connection from ::ffff:192.168.1.231
2022-01-27T18:45:31.743Z [ HYPERION] (DEBUG) (PriorityMuxer.cpp:165:registerInput()) Register new input 'Proto@::ffff:192.168.1.231/PROTOSERVER' with priority 100 as inactive
2022-01-27T18:45:31.744Z [ HYPERION] (DEBUG) (PriorityMuxer.cpp:271:setInputImage()) Priority 100 is now active
2022-01-27T18:45:31.744Z [ HYPERION] (DEBUG) (PriorityMuxer.cpp:374:setCurrentTime()) Set visible priority to 100
2022-01-27T18:45:31.745Z [ HYPERION] (DEBUG) (Hyperion.cpp:571:handleSourceAvailability()) priority[100], previousPriority[255]
2022-01-27T18:45:31.746Z [ HYPERION] (DEBUG) (Hyperion.cpp:582:handleSourceAvailability()) new source available -> Resume output processing and switch LED-Device on
2022-01-27T18:50:27.364Z [ PROTOSERVER] (DEBUG) (ProtoServer.cpp:68:newConnection()) Previous line repeats 1 times
2022-01-27T18:50:27.364Z [ PROTOSERVER] (DEBUG) (ProtoServer.cpp:68:newConnection()) New connection from ::ffff:192.168.1.233
2022-01-27T18:52:00.922Z [ WEBSOCKET] (DEBUG) (JsonAPI.cpp:122:handleInstanceSwitch()) Previous line repeats 1 times
2022-01-27T18:52:00.924Z [ WEBSOCKET] (DEBUG) (WebSocketClient.cpp:30:WebSocketClient()) New connection from ::ffff:192.168.1.6
2022-01-27T18:52:01.100Z [ WEBSOCKET] (DEBUG) (JsonAPI.cpp:122:handleInstanceSwitch()) Client '::ffff:192.168.1.6' switch to Hyperion instance 0
2022-01-27T18:52:05.373Z [ WEBSOCKET] (DEBUG) (WebSocketClient.cpp:30:WebSocketClient()) New connection from ::ffff:192.168.1.6
2022-01-27T18:52:05.575Z [ WEBSOCKET] (DEBUG) (JsonAPI.cpp:122:handleInstanceSwitch()) Client '::ffff:192.168.1.6' switch to Hyperion instance 0
2022-01-27T18:56:17.255Z [ PROTOSERVER] (DEBUG) (ProtoServer.cpp:68:newConnection()) New connection from ::ffff:192.168.1.231
2022-01-27T18:58:23.780Z [ WEBSOCKET] (DEBUG) (WebSocketClient.cpp:30:WebSocketClient()) New connection from ::ffff:192.168.1.6
2022-01-27T18:58:24.151Z [ WEBSOCKET] (DEBUG) (JsonAPI.cpp:122:handleInstanceSwitch()) Client '::ffff:192.168.1.6' switch to Hyperion instance 0
2022-01-27T18:58:24.494Z [ WEBSOCKET] (DEBUG) (WebSocketClient.cpp:30:WebSocketClient()) New connection from ::ffff:192.168.1.6
2022-01-27T18:58:24.880Z [ WEBSOCKET] (DEBUG) (JsonAPI.cpp:122:handleInstanceSwitch()) Client '::ffff:192.168.1.6' switch to Hyperion instance 0
2022-01-27T18:58:27.634Z [ WEBSOCKET] (DEBUG) (WebSocketClient.cpp:215:sendClose()) Send close to ::ffff:192.168.1.6: 1000 
2022-01-27T18:58:28.177Z [ WEBSOCKET] (DEBUG) (WebSocketClient.cpp:30:WebSocketClient()) New connection from ::ffff:192.168.1.6
2022-01-27T18:58:28.375Z [ WEBSOCKET] (DEBUG) (JsonAPI.cpp:122:handleInstanceSwitch()) Client '::ffff:192.168.1.6' switch to Hyperion instance 0
2022-01-27T18:59:09.308Z [ PROTOSERVER] (DEBUG) (ProtoServer.cpp:68:newConnection()) Previous line repeats 2 times
2022-01-27T18:59:09.308Z [ PROTOSERVER] (DEBUG) (ProtoClientConnection.cpp:76:disconnected()) Socket Closed
2022-01-27T18:59:09.309Z [ HYPERION] (DEBUG) (PriorityMuxer.cpp:292:clearInput()) Removed source priority 100
2022-01-27T18:59:09.309Z [ HYPERION] (DEBUG) (PriorityMuxer.cpp:374:setCurrentTime()) Set visible priority to 255
2022-01-27T18:59:09.309Z [ HYPERION] (DEBUG) (Hyperion.cpp:571:handleSourceAvailability()) priority[255], previousPriority[100]
2022-01-27T18:59:09.310Z [ HYPERION] (DEBUG) (Hyperion.cpp:574:handleSourceAvailability()) No source left -> Pause output processing and switch LED-Device off

**Comment by Paulchen-Panther** [comment]
There it seems to be a hardware problem. No problem on the part of the grabber if it works with LAN cable. Therefore I close here. Thanks for your understanding.

**Comment by A123670011** [comment]
I changed the router to another ASUS AX88U Router. 
But the issue does still exist:
The connection to the hyperion server stops after a few seconds, reconnects for a few seconds and so on. 

I installed the android hyperion grabber at my smartphone and here the connection works fine in the same WiFi and with the same router. The connection does also work if I use a LAN connection instead of wifi connection between the router and the Nvidia Shield. 

So I think that the problem does only exist if I connect a Nvidia Shield via WiFi to an ASUS router.
